appinventor/blocklyeditor/src/blocks/math.js

```diff
@@ -312,6 +312,44 @@ Blockly.Blocks['math_power'] = {
   typeblock: [{translatedName: Blockly.Msg.LANG_MATH_ARITHMETIC_POWER}]
 };
 
+Blockly.Blocks['math_bitwise_and'] = {
+    // Basic arithmetic operator.
```

Comment:
Indentation here is 4 instead of 2. You will need to update your JavaScript style settings in addition to the Java ones.